### PR TITLE
Fixs build failure due to status update in old code base

### PR DESCRIPTION
**Pull request by bharathappali**
Signed-off-by: bharathappali <abharath@redhat.com>

### CODE REVIEW

**Comment by bharathappali** [comment]
[status_issue_build.log](https://github.com/kruize/autotune/files/9575420/status_issue_build.log)
Checked if the build is working and updating comment with the log

docker image: `bharathappali/exman:status-update`

**Comment by bharathappali** [comment]
@dinogun The PR is ready for review

**Comment by dinogun** [comment]
What testing has been done with these changes. Can you post the results of the loop test here

File: src/main/java/com/autotune/common/annotations/json/AutotuneJSONExclusionStrategy.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.autotune.common.annotations.json;
```
Comment (dinogun) [inline src/main/java/com/autotune/common/annotations/json/AutotuneJSONExclusionStrategy.java:1]: LICENSE header missing

File: src/main/java/com/autotune/common/annotations/json/Exclude.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package com.autotune.common.annotations.json;
```
Comment (dinogun) [inline src/main/java/com/autotune/common/annotations/json/Exclude.java:1]: LICENSE header missing

**Comment by bharathappali** [comment]
Updated result of the autotune log which mentions successful completion of loop and the JSON excludes flags

[status_issue_build.log](https://github.com/kruize/autotune/files/9581854/status_issue_build.log)

Docker image : ` bharathappali/exman:status-update`

File: src/main/java/com/autotune/common/annotations/json/AutotuneJSONExclusionStrategy.java (hunk 1 above)
Comment (bharathappali) [reply src/main/java/com/autotune/common/annotations/json/AutotuneJSONExclusionStrategy.java:16]: Thanks for the review @dinogun . Added the LICENSE Header

File: src/main/java/com/autotune/common/annotations/json/Exclude.java (hunk 2 above)
Comment (bharathappali) [reply src/main/java/com/autotune/common/annotations/json/Exclude.java:16]: Thanks for the review @dinogun . Added the LICENSE Header

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**